docker/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+GIT_REF=${GIT_REF:-master}
+
+# Use Git respository as a source when there's no `/airnode` directory mounted
+if [ -d /airnode ]; then
+  # Efficient way to copy the whole directory with a lot of small files
+  # Keeping any local code changes but excluding platform-specific files (dependencies, builds, etc.)
+  rsync -a --include packages/airnode-protocol/deployments --exclude node_modules --exclude build --exclude dist --exclude coverage --exclude artifacts --exclude cache --exclude .webpack --exclude deployments /airnode/ /build
```

Comment:
I presume this will not include not-commited changes, will it? I wouldn't want to lose that as that's the point of a local build - you do some changes and you want to test whether it works in the container, not necessarily committing it first. I'll check and try to figure out something else if that's the case.

Reply:
True. It will not include uncommitted changes. 

Btw. this command is fast for me as is. I just fear that we will have a new ignored folder and forget to add it here. But feel free to merge this as is.

Reply:
I want to avoid that as well. I'll try to reuse the `.gitignore`.